newsfragments/984.feature.rst

```diff
@@ -0,0 +1 @@
+Use __slots__ in more places internally, which should make Trio slightly faster.
```

Comment:
Since this is restructured text, I think you need to add double-backticks around `__slots__`, like:

```rst
Use ``__slots__`` in more places internally
```